app/controllers/users_controller.rb

```diff
@@ -18,9 +18,29 @@ def create
     end
   end
 
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def edit
+    @user = User.find(params[:id])
```

Comment:
Please extract this line into a `before_filter` that is used by all three actions.